src/views/List.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="list">
+    <h1>List All Podcasts</h1>
+    <list-podcast :list="podcasts" @search-track="filterPodcasts" :footer="false" width="90%"/>
+  </div>
+</template>
+
+<script>
+import api from '@/api';
+import { getRandomPodcastCategory } from '@/utils';
+import ListPodcast from '@/components/list-podcast/list-podcast.vue';
+
+export default {
+  name: 'List',
+  data() {
+    return {
+      podcasts: [],
+    };
+  },
+  components: {
+    ListPodcast,
+  },
+  methods: {
+    filterPodcasts(term) {
+      api.search(term).then((res) => {
```

Comment:
Please lets use await 
`    async filterPodcasts(term) {
      this.podcasts = await api.search(term);
    },`